test/Sema/access-level-and-non-resilient-import.swift

```diff
@@ -0,0 +1,51 @@
+// RUN: %empty-directory(%t)
+// RUN: split-file %s %t
+
+/// Build the libraries.
+// RUN: %target-swift-frontend -emit-module %t/DefaultLib.swift -o %t
+// RUN: %target-swift-frontend -emit-module %t/PublicLib.swift -o %t
+// RUN: %target-swift-frontend -emit-module %t/PackageLib.swift -o %t
+// RUN: %target-swift-frontend -emit-module %t/InternalLib.swift -o %t
+// RUN: %target-swift-frontend -emit-module %t/FileprivateLib.swift -o %t
+// RUN: %target-swift-frontend -emit-module %t/PrivateLib.swift -o %t
+
+/// A resilient client will error on public imports.
+// RUN: %target-swift-frontend -typecheck %t/Client_Swift5.swift -I %t \
+// RUN:   -enable-library-evolution -swift-version 5 \
+// RUN:   -enable-experimental-feature AccessLevelOnImport -verify
+// RUN: %target-swift-frontend -typecheck %t/Client_Swift6.swift -I %t \
+// RUN:   -enable-library-evolution -swift-version 6 \
+// RUN:   -enable-experimental-feature AccessLevelOnImport -verify
+
+/// A non-resilient client doesn't complain.
+// RUN: %target-swift-frontend -typecheck %t/Client_Swift5.swift -I %t \
+// RUN:   -swift-version 5 \
+// RUN:   -enable-experimental-feature AccessLevelOnImport
+// RUN: %target-swift-frontend -typecheck %t/Client_Swift6.swift -I %t \
+// RUN:   -swift-version 6 \
```

Comment:
I don't see test failures so maybe I'm wrong about this but I thought tests that use `-swift-version 6` require asserts currently.

Reply:
Good point, thanks! I believe PR testing only uses asserts mode so it would have failed after merging.